nav2_smac_planner/src/smoother.cpp

```diff
@@ -136,7 +137,7 @@ bool Smoother::smoothImpl(
         "Smoothing time exceeded allowed duration of %0.2f.", max_time);
       path = last_path;
       updateApproximatePathOrientations(path, reversing_segment);
-      return false;
+      throw nav2_core::SmootherTimedOut("Smoothing time exceed allowed duration");
```

Comment:
This is internal to the path planner not a separate smoother plugin - this should be handled by the planner exceptions / failure codes. Revert.